Google.PowerShell.Tests/Common/TestActiveUserCredential.cs

```diff
@@ -0,0 +1,138 @@
+﻿using NUnit.Framework;
+using Google.Apis.Util;
+using Google.PowerShell.Common;
+using System;
+using System.Reflection;
+using System.Threading;
+
+namespace Google.PowerShell.Tests.Common
+{
+    [TestFixture]
+    /// <summary>
+    /// Assumes that the Cloud SDK is installed on the local machine and initialized.
+    /// 
+    /// </summary>
+    internal class TestActiveUserCredential
+    {
+        private PropertyInfo TokenProperty = typeof(ActiveUserCredential).GetProperty(
```

Comment:
To be consistent with the .NET naming guidelines, this should be `tokenProperty` (lowercase T). Same with `cancelToken` below.